### PR TITLE
UI fixes - Bugs on the PointInformation and several other

### PR DESCRIPTION
**Pull request by UO283069**
Added a few changes to fix bugs on the UI.

### CODE REVIEW

**Review by UO283428 — APPROVED** [review]

**Review by Davidmc07 — APPROVED** [review]

File: webapp/src/components/social/UserPlaceHolder.tsx [hunk 1]
```diff
@@ -45,26 +45,42 @@ export default class UserPlaceHolder extends React.Component<UserPlaceHolderProp
         });
     }
 
+    private shortText(text: string | null) {
+        if (text === null) {
+            return "Unknwon friend";
```
Comment (uo282892) [inline webapp/src/components/social/UserPlaceHolder.tsx:50]: "Unknown"

**Review by uo282892 — COMMENTED** [review]
Check the typo

File: webapp/src/components/social/UserPlaceHolder.tsx (hunk 1 above)
Comment (UO283428) [reply webapp/src/components/social/UserPlaceHolder.tsx:50]: I will change it right away after the merge!

**Review by uo282892 — COMMENTED** [review]
I was going to approve that once you changed the typo.

Just fix that and it is all fine

**Pull request merged**